src/Component/Menu/User/User.tsx

```diff
@@ -40,9 +40,11 @@ export const User: React.FC<UserProps> = (props) => {
         setInfoVisible(true);
         break;
       case 'settings':
+        // TODO: Fix settings for non keycloak setups
```

Comment:
This is probably part of a separate PR?

Reply:
The TODO will be hopefully removed in an upcoming PR. If not we do not forget about it. :wink: